api/v1alpha1/nodepool_types.go

```diff
@@ -274,7 +274,26 @@ type RollingUpdate struct {
 
 // InPlaceUpgrade specifies an upgrade strategy which upgrades nodes in-place
 // without any new nodes being created or any old nodes being deleted.
-type InPlaceUpgrade struct{}
+type InPlaceUpgrade struct {
+	// MaxUnavailable is the maximum number of nodes that can be unavailable
+	// during the update.
+	//
+	// Value can be an absolute number (ex: 5) or a percentage of desired nodes
+	// (ex: 10%).
+	//
+	// Absolute number is calculated from percentage by rounding down.
+	//
+	// Defaults to 0.
```

Comment:
Zero is blocking state right, so why default to zero?
Wouldn't we want to default to 1?

Reply:
So, this is a relic from the MCO I think. In the MCO, you can set it to 0, but it will always get set to 1 if it is 0.

I can change this here though, no reason to keep it that way